test_setup.alerts_logger.yml

```diff
@@ -0,0 +1,36 @@
+---
+# ========================================
+#  QE configuration for alerts logger
+# ========================================
+#
+# This playbook serves as setup of alerts logger on tendrl server machine.
+# It periodically logs alerts from tendrl api based on tendrl user.
+
+- hosts: usm_server
+  remote_user: root
+  vars:
+    tendrl_user: "admin"
+    tendrl_password: "adminuser"
+
+  tasks:
+    - name: Create alerts_logger python script based on a template
+      template:
+        dest: /usr/bin/alerts_logger_{{ tendrl_user }}.py
+        src: alerts_logger.py.j2
+        force: yes
+        mode: "+x"
+
+    - name: Create service file based on a template
+      template:
+        dest: /lib/systemd/system/alerts_logger_{{ tendrl_user }}.service
```

Comment:
It would be better to put it into `/etc` instead, as `/lib/systemd` contains the packaged defaults.